src/clj_http_ssrf/core.clj

```diff
@@ -1,15 +1,35 @@
 (ns clj-http-ssrf.core
   (:require [clj-http.client :as client]
-            [inet.data.ip :as ip])
-  (:import [java.net InetAddress]))
-
+            [clj-http.util :refer [opt]]
+            [inet.data.ip :as ip]
+            [clojure.stacktrace :refer [root-cause]])
+  (:import (java.net InetAddress UnknownHostException)))
 
 (defn- make-response
   [status headers body]
   {:status status
    :headers headers
    :body body})
 
+(defn- get-host-address
+  ;; Passing in parsed-url since it was already computed
+  [req parsed-url]
+  (let [server-name (:server-name parsed-url)]
+    (try
+      (.getHostAddress (InetAddress/getByName server-name))
+      (catch Exception e
+        (if (or (not= (type (root-cause e)) UnknownHostException)
+                (not (opt req :ignore-unknown-host)))
+          (throw (root-cause e)))))))
+
+(defn- parse-url
+  [req]
+  (let [url        (:url req)
+        parsed-url (client/parse-url url)]
+    {:url url
```

Comment:
Similar to aligning let. map could also be aligned. Please refer : https://github.com/bbatsov/clojure-style-guide#vertically-align-let-and-map . 

I found a way to do this in Emacs. Select the map and hit `C-c SPC` to align the map. Link : https://github.com/clojure-emacs/clojure-mode#vertical-alignment